modules/nf-core/arriba/download/main.nf

```diff
@@ -9,6 +9,7 @@ process ARRIBA_DOWNLOAD {
 
     input:
     val(genome)
```

Comment:
If this parameter is not used, then I suggest removing it unless it's used for naming (in which case use `prefix` as other modules)